### DISCUSSION TITLE
Can I access the render system manually?

### DISCUSSION THREAD

**Discussion by DasLixou** [bevyengine/bevy#5907] [Q&A]
Edit: Is there a way to manually draw multiple sprites manually to the render system?

Old: Hello! Im new to rust and bevy and wanna ask what the simplest method for drawing multiple sprites is. I want to create a chunk system where i can just draw a chunk with data at a special position and thought of putting every texture that should be rendered in like a batch. Whats the bevyiest way to do that? thanks :)

**Comment by brandon-reinhart** [comment]
You might start by looking at the sprite sheet example: https://github.com/bevyengine/bevy/blob/main/examples/2d/sprite_sheet.rs

When you say "chunk" I wonder if you mean something like a chunked tilemap? In which case, you might look into bevy_ecs_tilemap: https://crates.io/crates/bevy_ecs_tilemap

Unfortunately, I don't understand your goals well enough to give clearer advice.

**Reply by DasLixou** [reply]
So i just wonder if there is some render batch methods that I can call manually to implement my own render technics
